src/App.js

```diff
@@ -1,29 +1,69 @@
 import './App.css';
+import '../node_modules/bootstrap-icons/font/bootstrap-icons.css'
+import {useState} from 'react'
+import {render} from 'react-dom'
+let list=[];
+const input=document.getElementById('input');
+
+let btnFunction='new';
+let clickAction={
+  'edit':function(e){
+    e.preventDefault();
+    console.log(e.target.attributes.index.value)
+  },
+'delete':function(e){
+  e.preventDefault();
+  console.log(e.target.attributes.index.value)
+
+}
+}
+
+function Itemslist (props){
+  let itemsarray=props.list
+  if(itemsarray.length>0){
+  const items=itemsarray.map((item,index)=>(<li className="list-group-item d-flex justify-content-between align-items-center" >
+  <span className="title">{item}</span>
+  <span>
+      <a href="#" onClick={props.onClick.edit} data-edit ><i className="bi bi-pencil-square blue" index={index} ></i></a>
+      <a href="#" onClick={props.onClick.delete}  data-delete><i className="bi bi-x-circle red" index={index}></i></a>
+  </span>
+</li>))
+return items
+  }
+  return ""
+}
+
+
+
 
 function App() {
+  console.log("render")
+  var [itemsarray,setItemsarray]=useState(list);
+  let value;
+  const handleChange=(e)=>{
+    e.preventDefault();
+value=e.target.value;
+console.log(value,1);
```

Comment:
when you are done with the debugging and testing of your code, remember to remove any `console.log`